frog/imports/client/Wiki/index.js

```diff
@@ -140,9 +141,10 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
 
     window.wikiDoc = this.wikiDoc;
     if (!this.props.embed) {
-      Mousetrap.bindGlobal('ctrl+n', () =>
-        this.setState({ createModalOpen: true })
-      );
+      Mousetrap.bindGlobal('ctrl+n', () => {
+        if (!this.state.settings?.noNewPage && !this.state.isOwner)
```

Comment:
`noNewPage` is not defined in `WikiSettingsT`

Reply:
I think this was an inadvertent error